src/main/webapp/readfile.html

```diff
@@ -15,57 +15,25 @@ <h2>Read Configuration</h2>
       <a href="deletefile.html">Delete Configuration</a>
     </div>
     <div>
-    <form id="config-search" action="/BillingConfig" method="get">
+    <form id="config-search" action="/BillingConfig">
```

Comment:
I think this should still use the get method, but I'm not certain

Reply:
It should be a get method by default when fetching. But I can change it.